src/main/java/uk/ac/cam/cl/dtg/segue/api/AbstractSegueFacade.java

```diff
@@ -327,4 +327,9 @@ public static boolean isUserTeacherOrAbove(final UserAccountManager userManager,
             throws NoUserLoggedInException {
         return !userManager.checkUserRole(userDTO, Collections.singletonList(Role.STUDENT));
     }
+
+    public static boolean isUserTeacher(final UserAccountManager userManager, final RegisteredUserDTO userDTO)
```

Comment:
Am I right in thinking we do not need this? Generally we have things admin can do, things staff can do, things teachers or above can do, and things anyone can do. I don't think we want anything that explicitly only applies to teachers except by exclusion. And I think the place that calls this doesn't need it either?